### PR TITLE
Move AU Core MedicationStatement out of current R1 (https://jira.hl7.org/browse/FHIR-45052)

### PR DESCRIPTION
**Pull request by dbojicic**
Fixes https://jira.hl7.org/browse/FHIR-45052. 

Changes to move AU Core MedicationStatement out of current R1: 
- removed AU Core MedicationStatement StructureDefinition, examples and associated intro and notes markdown files
- removed terminology related to MedicationStatement from the terminology page
- in General Requirements, removed AU Core MedicationStatement from the tables in Must Support section, and replaced MedicationStatement examples with MedicationRequest 
- removed AU Core MedicationStatement from comparison with other specs, AU Core Client and AU Core Server CapabilityStatements
-  in General Guidance page, moved AU Core MedicationStatement from the list of profiles defined in AU Core to the list of future releases
- Added an entry in Future of AU Core 
- Removed hyperlinks to AU Core MedicationStatement profile but kept the mappings in AUCDI page (noting that other mappings (ie AU Core MedicationRequest) are unchanged)
- Updated change log

### CODE REVIEW

**Comment by dbojicic** [comment]
@dt-r, please hold off the review until the changes from #143 are brought in.

**Comment by dbojicic** [comment]
@dt-r, this PR can be reviewed now.

File: input/pagecontent/future.md [hunk 1]
```diff
@@ -29,4 +29,9 @@ The approach to develop and mature AU Core is outlined below making reference to
     <img src="maturing.png" alt="Maturing the AU Core baseline" style="width:80%"/>
   </div>
 *Figure 3: Maturing the AU Core baseline*
-<br/>
\ No newline at end of file
+<br/>
+
+### AU Core Profiles Roadmap
```
Comment (dt-r) [inline input/pagecontent/future.md:34]: Heading should be "Future Candidate Requirements Under Consideration"
Have a sentence under the heading. "The following items are under consideration to add AU Core."

File: input/pagecontent/future.md [hunk 2]
```diff
@@ -29,4 +29,9 @@ The approach to develop and mature AU Core is outlined below making reference to
...
   </div>
 *Figure 3: Maturing the AU Core baseline*
-<br/>
\ No newline at end of file
+<br/>
+
+### AU Core Profiles Roadmap 
+
+#### AU Core MedicationStatement 
+The development of the AU Core MedicationStatement profile is planned for the AU Core R2 scope, following the TDG's strategic decision to prioritise the AU Core MedicationRequest profile for R1. Should time allow, the TDG plans to work on the AU Core MedicationStatement within R1 as well.
```
Comment (dt-r) [inline input/pagecontent/future.md:37]: Suggest instead
Development and testing of AU Core MedicationStatement profile to support AUCDI Medication Use Statement is planned for the AU Core R2 scope.

File: input/pagecontent/aucdi.md [hunk 3]
```diff
@@ -136,61 +136,61 @@ The table below shows the relationship between the AUCDI Data Groups and Element
   <tr>
     <td rowspan="10">Medication use statement</td>
     <td colspan="2">Medication name</td>
-    <td><a href="StructureDefinition-au-core-medicationstatement.html">AU Core MedicationStatement</a> | <a href="StructureDefinition-au-core-medication.html">AU Core Medication</a> | <a href="StructureDefinition-au-core-medicationrequest.html">AU Core MedicationRequest</a></td>
+    <td>AU Core MedicationStatement | <a href="StructureDefinition-au-core-medication.html">AU Core Medication</a> | <a href="StructureDefinition-au-core-medicationrequest.html">AU Core MedicationRequest</a></td>
     <td>MedicationStatement.medication[x] | MedicationRequest.medication[x] | Medication.code</td>
     <td>Feedback is requested on the appropriateness of using either the MedicationStatement resource or MedicationRequest resource (or both) to represent a medication use statement. See the discussion in chat.fhir.org: <a href="https://chat.fhir.org/#narrow/stream/179173-australia/topic/MedicationRequest.20or.20MedicationStatement">https://chat.fhir.org/#narrow/stream/179173-australia/topic/MedicationRequest.20or.20MedicationStatement</a>.</td>
```
Comment (dt-r) [inline input/pagecontent/aucdi.md:141]: Add the text in the Comment column:
Development and testing of AU Core MedicationStatement profile is planned for AU Core R2.

File: input/pagecontent/changes.md [hunk 4]
```diff
@@ -13,3 +13,4 @@ This change log documents the significant updates and resolutions implemented fr
   - Procedure.recorder [FHIR-45015](https://jira.hl7.org/browse/FHIR-45015)
   - Procedure.asserter [FHIR-45016](https://jira.hl7.org/browse/FHIR-45016)
   - Procedure.bodySite and added profile-specific implementation guidance on including body site information [FHIR-45017](https://jira.hl7.org/browse/FHIR-45017).
+- Removed AU Core MedicationStatement and added narrative guidance in Future of AU Core on the future of the profile [FHIR-45052](https://jira.hl7.org/browse/FHIR-45052)
```
Comment (dt-r) [inline input/pagecontent/changes.md:16]: change "added narrative guidance in Future of AU Core on the future of the profile "
to be "and added to Future of AU Core that AU Core MedicationStatement is planned for AU Core R2."

**Review by dt-r — CHANGES_REQUESTED** [review]
Few tweaks requested

File: input/pagecontent/aucdi.md (hunk 3 above)
Comment (dbojicic) [reply input/pagecontent/aucdi.md:141]: Done

File: input/pagecontent/changes.md (hunk 4 above)
Comment (dbojicic) [reply input/pagecontent/changes.md:16]: Done

File: input/pagecontent/future.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/future.md:37]: Done

File: input/pagecontent/future.md (hunk 1 above)
Comment (dbojicic) [reply input/pagecontent/future.md:34]: Done

File: input/pagecontent/future.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/future.md]: still says the words scope after R2. But we can fix this later.

**Review by dt-r — CHANGES_REQUESTED** [review]
1. Missing change to prove white list to remove MedicationStatement
2. Create a GitHub for a later ticket to add more clarification on what remove of the profile means in terms of normative changes

**Comment by dbojicic** [comment]
@dt-r 
1. Removed MedicationStatement from prov-white-list
2. Created #146

**Review by dt-r — APPROVED** [review]

**Pull request merged**